spec/models/subscription_spec.rb

```diff
@@ -3,16 +3,15 @@
 require 'rails_helper'
 
 RSpec.describe Subscription, type: :model do
+  let(:plan) { create(:plan) }
```

Comment:
RSpec/EmptyLineAfterFinalLet: Add an empty line after the last `let`. (https://rspec.rubystyle.guide/#empty-line-after-let, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/EmptyLineAfterFinalLet)